src/components/card/card.pw.tsx

```diff
@@ -0,0 +1,353 @@
+import { expect, test } from "@playwright/experimental-ct-react17";
+import React from "react";
+import "../../../playwright/components";
+import {
+  card,
+  columnCard,
+  draggableCard,
+  draggableContainer,
+  footerCard,
+} from "../../../playwright/components/card/index";
+import { CHARACTERS, SIZE } from "../../../playwright/support/constants";
+import {
+  assertCssValueIsApproximately,
+  checkAccessibility,
+} from "../../../playwright/support/helper";
+import { Card, CardProps } from "../../../src/components/card";
+import {
+  DifferentCardFooterPadding,
```

Comment:
Why are these all from lines 18-33 in upper camel case? I can see that they're like that in the Cypress tests but they should be lower camel case to follow our current standard. Can you change them please while you're in the files?

Reply:
These are custom components imported from the stories files/test-pw files. They need to pascal case. We do this in the other Playwright files too. [Here is Batch-Selection](https://github.com/Sage/carbon/blob/master/src/components/batch-selection/batch-selection.pw.tsx#L4) and the [custom component in the test-pw file here](https://github.com/Sage/carbon/blob/master/src/components/batch-selection/components.test-pw.tsx#L6) too.